frontend/src/views/activity/Category.vue

```diff
@@ -1,109 +0,0 @@
-<template>
```

Comment:
ok, why does github think you deleted the file and then added it again?

Reply:
I don't know. I even commited the move on its own, to prevent this. So frustrating.